Install/InstallVilo3D.md

```diff
@@ -0,0 +1,319 @@
+
```

Comment:
I get it now: this was previously in UDV repository...

Reply:
Yes; I updated it to add the server - side installation and moved it from UDV to RICT.

Reply:
Nice.